wdl/model/draft3/src/main/scala/wdl/model/draft3/elements/WorkflowOutputDeclarationElement.scala

```diff
@@ -0,0 +1,5 @@
+package wdl.model.draft3.elements
+
+final case class WorkflowOutputDeclarationElement(valueType: String, name: String, value: String) extends LanguageElement {
+  override def children: Seq[LanguageElement] = Seq.empty
```

Comment:
Leave them for now, but as a heads up I'm going to remove this "children" field soon. 

It's extra typing for us and I'm not convinced we need it. We certainly don't need it right now!